src/main/kotlin/StringCalculator.kt

```diff
@@ -0,0 +1,73 @@
+import CheckString.checkOperator
+import java.util.Scanner
+
+class StringCalculator {
+    private var result: Int = 0
+    val checkString = CheckString
+
+    fun readEquation() {
+        println("계산할 식을 입력하세요")
+        val scanner = Scanner(System.`in`)
+        val input = scanner.nextLine()
+        splitEquation(input)
+    }
+
+    fun splitEquation(input: String) {
+        checkString.checkBlank(input)
+        val splitString: List<String> = input.split(" ")
+        calculate(splitString)
+    }
+
+    fun calculate(splitString: List<String>): Boolean {
+        val finalresultNumber = setUpCalculate(splitString)
+        if (checkOperator(splitString) == false) {
+            return false
+        }
```

Comment:
```suggestion
        val finalResultNumber = setUpCalculate(splitString)
        if (!checkOperator(splitString)) {
            return false
        }
```
요렇게 쓸수있어! 카멜케이스 지키도록 노력해보자~~

Reply:
앗 카멜케이스 이런 실수를 ㅋㅋㅋ감사합니당